vaadin-ordered-layout-flow-demo/src/main/java/com/vaadin/flow/component/orderedlayout/demo/OrderedLayoutView.java

```diff
@@ -151,22 +142,16 @@ private void createHorizontalLayoutWithDefaultAlignment() {
         component2.getElement().getStyle().set("fontSize", "24px");
         component3.getElement().getStyle().set("fontSize", "9px");
 
-        Div buttons = new Div();
-        buttons.add(createAlignmentButton(layout, "align-start-button",
-                FlexComponent.Alignment.START));
-        buttons.add(createAlignmentButton(layout, "align-end-button",
-                FlexComponent.Alignment.END));
-        buttons.add(createAlignmentButton(layout, "align-center-button",
-                FlexComponent.Alignment.CENTER));
-        buttons.add(createAlignmentButton(layout, "align-stretch-button",
-                FlexComponent.Alignment.STRETCH));
-        buttons.add(createAlignmentButton(layout, "align-baseline-button",
-                FlexComponent.Alignment.BASELINE));
+        RadioButtonGroup<FlexComponent.Alignment> alignments = new RadioButtonGroup<>();
+        alignments.setItems(FlexComponent.Alignment.values());
+        alignments.setValue(FlexComponent.Alignment.CENTER);
+        alignments.setId("horizontal-layout-alignment-radio-button");
+        alignments.addValueChangeListener(event -> layout.setDefaultVerticalComponentAlignment(event.getValue()));
 
         layout.setId("layout-with-alignment");
 
         addCard("HorizontalLayout", "HorizontalLayout with general alignment",
```

Comment:
The AUTO button can be removed as it is working neither in this nor in VerticalLayout with general alignment demo

Reply:
Haven't changed this part, because getting all values from `ENUM`.